source/about-mongosync.txt

```diff
@@ -162,5 +162,6 @@ Learn More
 
 To learn more about ``mongosync``, see:
 
-- :ref:`mongosync <c2c-mongosync>`
+- :ref:`mongosync Behavior <c2c-mongosync-behavior>`
+- :ref:`mongosync Options <c2c-mongosync-options>`
```

Comment:
Can you add staging links for all of the pages that you've edited to this PR?

Reply:
Just did, sorry about that!